api/app/GraphQL/Mutations/SubmitApplication.php

```diff
@@ -120,7 +38,7 @@ public function __invoke($_, array $args)
         $application->submitted_at = $dateNow;
         $application->pool_candidate_status = ApiEnums::CANDIDATE_STATUS_NEW_APPLICATION;
         $application->expiry_date = $expiryDate;
-        $application->signature = $signature;
+        $application->signature = $args['signature'];
```

Comment:
Makes sense to me! Updated with [move application assignments up to group together](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5297/commits/4f151c2ec1ffd2263d1a4b4ca4a2ea35f1a1f846)

Reply:
Actually, dang. This breaks the validation because it is populating values we are checking against so it has to be after validation.

Reverted in [Revert "move application assignments up to group together"](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5297/commits/153c2466131a682dbbbe7362606f9d0c66f6e31b)

Then, removed duplicate assignment in [remove duplicate signature assignment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5297/commits/e8ac5c8ed03745b370b8e0bf1eb475e190a84995)

Reply:
Ah makes sense.  It was fine as it was before.